parsley-debug/shared/src/main/scala/parsley/debug/internal/DivergenceContext.scala

```diff
@@ -17,7 +17,7 @@ import parsley.internal.deepembedding.frontend.LazyParsley
 private [parsley] final class DivergenceContext {
     case class CtxSnap(pc: Int, instrs: Array[_], off: Int, regs: List[AnyRef])
     case class HandlerSnap(pc: Int, instrs: Array[_])
-    private case class Snapshot(name: String, internalName: String, ctxSnap: CtxSnap, handlerSnap: Option[HandlerSnap], children: mutable.ListBuffer[Snapshot]) {
+    private case class Snapshot(name: String, internalName: String, isIterative: Boolean, ctxSnap: CtxSnap, handlerSnap: Option[HandlerSnap], children: mutable.ListBuffer[Snapshot]) {
```

Comment:
This actually doesn't need it: for now `DivergenceContext`, and the `Divergence` internal frontend/backend nodes, can just ignore the field. (the factory implementation will just ignore it). If I want it later, I'll add it then, for now it's just increasing the memory usage